core/controlplane/config/token_config.go

```diff
@@ -152,6 +153,12 @@ func (r *EncryptedAuthTokensOnDisk) Compact() (*CompactAuthTokens, error) {
 
 func ReadOrEncryptAuthTokens(dirname string, encryptor CachedEncryptor) (*EncryptedAuthTokensOnDisk, error) {
 	authTokenPath := filepath.Join(dirname, "tokens.csv")
+
+	// Auto-creates the auth token file, useful for those coming from previous versions of kube-aws
+	if _, err := os.Stat(authTokenPath); os.IsNotExist(err) {
+		os.OpenFile(authTokenPath, os.O_RDONLY|os.O_CREATE, 0600)
```

Comment:
Argh, forgot to close it. 😓 

@mumoshu just to let you know, feel free not to merge my PRs unless they are 100% according to your standards regarding code style, quality, and so on; just give me your feedback and I'll try to address them ASAP.